.github/workflows/deployment.yml

```diff
@@ -6,6 +6,24 @@ on:
       - main
 
 jobs:
+  prepare-database:
```

Comment:
Any reason why this job gets special treatment?

Reply:
As we discussed - it runs in parallel with the build. Both build and db now dependant on the secrets.